test/functional/feature_assumeutxo.py

```diff
@@ -152,6 +148,38 @@ def test_invalid_mempool_state(self, dump_output_path):
 
         self.restart_node(2, extra_args=self.extra_args[2])
 
+    def test_snapshot_in_a_divergent_chain(self, dump_output_path):
+        # First rollback node2's chain to the pregenerated one, up to height 199
+        node = self.nodes[2]
+        current_height = node.getblockcount()
+        base_height = 199
```

Comment:
is this not the same as `START_HEIGHT`?

Reply:
Yes, good catch. This line was removed since the iteration is no longer needed based on your other comment below.